src/core/render/index.js

```diff
@@ -242,6 +242,26 @@ export function Render(Base) {
         el.setAttribute('href', nameLink[match]);
       }
     }
+
+    #renderSkipLink(vm) {
+      const { skipLink } = vm.config;
+
+      if (skipLink !== false) {
+        const el = dom.getNode('#skip-to-content');
+        const text =
+          typeof skipLink === 'string'
+            ? skipLink
+            : skipLink?.[vm.route.path] || 'Skip to main content';
```

Comment:
This is a property access, but should it be a regex? Can this currently fail in some cases?

Reply:
Documentation here available in the preview: 

https://docsify-preview-git-494-skip-link-docsifyjs.vercel.app/#/configuration?id=skiplink